backend/isoko-backend/src/handlers/review/post-review.js

```diff
@@ -0,0 +1,68 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const docClient = new dynamodb.DocumentClient();
+const _ = require('lodash');
+const { BUSINESS_TABLE } = require('../../constants');
+
+/**
+ * HTTP post method that creates a review for the specified business.
+ */
+exports.postReviewHandler = async (event) => {
+   if (event.httpMethod !== 'POST') {
+      throw new Error(
+         `postReviewHandler only accept POST method, you tried: ${event.httpMethod}`
+      );
+   }
+
+   console.info('received:', event);
+
+   const { businessId } = event.pathParameters;
+
+   if (businessId == null) {
```

Comment:
```suggestion
   if (!businessId) {
```

Just to be safe I think we should just check if businessId is undefined, this will still go into the if statement if businessId is null, but this will also catch empty strings or undefined as well